R/DataSet.R

```diff
@@ -545,7 +545,7 @@ get_overview.DataSet <- function(ds, ...) {
   runs <- ncol(data)
 
   budget <- max(attr(ds, 'maxRT'))
-  if (!is.null(ds$RT)) {
+  if (!is.null(ds$RT) && length(ds$RT) > 0) {
```

Comment:
In which case `ds$RT` is a zero-length matrix? should it be `NULL` for Nevergrad?

Reply:
This is just in case this happens, I had some weird data (I think it included some nan-values) where this occurred, so this is just to prevent those issues.